docs/get-started.md

```diff
@@ -98,8 +98,15 @@ var client = new WebTorrent()
 var magnetURI = 'magnet: ...'
 
 client.add(magnetURI, { path: '/path/to/folder' }, function (torrent) {
+  var isDone = false
+
   torrent.on('done', function () {
-    console.log('torrent download finished')
+    console.log('\ntorrent download finished')
+    isDone = true
+  })
+
+  torrent.on('download', function (bytes) {
+    if (!isDone) process.stdout.write(`\rtotal: ${torrent.length}  progress: ${(client.progress * 100).toFixed(2)}%  ${Math.round(client.downloadSpeed / 1024).toFixed(1)} KB/s  downloading...`)
```

Comment:
I don't think there is any need in adding this condition.

Every time the event `download` is called, is because you have downloaded something.

Reply:
I've tested and it does seem that without `isDone` you will continue to get a few `download` calls showing the progress after `done` is called. Is this what you are experiencing as well?

Reply:
I've tested and it does seem that without `isDone` you will continue to get a few `download` calls showing the progress after `done` is called. Is this what you are experiencing as well?

Reply:
aha, at first I didn't have the `isDone`, and after the "Done" message is printed, then there is one additional progress message that got printed that messed up the screen.  If the event occurs such that the progress always happens first, and then the `done` occurs, then there is no need for the `isDone` flag.

Reply:
We could defer done to next event loop to solve this

Reply:
Yep, I agree @DiegoRBaquero